src/Tribe/Dependency.php

```diff
@@ -175,6 +195,60 @@ public function has_requisite_plugins( $plugins_required = array() ) {
 			return true;
 		}
 
+		/**
+		 * Retrieves Registered Plugin by Class Name from Array
+		 *
+		 * @return array
+		 */
+		public function get_registered_plugin( $class ) {
+			$plugins = $this->registered_plugins;
+
+			return isset( $plugins[ $class ] ) ? $plugins[ $class ] : array();
+		}
+
+		/**
+		 * gets all dependencies or single class requirements
+		 * if parent, co, add does not exist use array as is
+		 * if they do exist check each one in turn
+		 *
+		 */
+		public function dependency_checker( $dependencies = array(), $plugin ) {
```

Comment:
This method would benefit from being renamed away from a noun since it performs an action and returns a boolean. Perhaps `has_valid_dependencies()`?

I wonder if it would be more intuitive to swap the order of the parameters in this method, like: `( $plugin, $dependencies = array() )`. It feels a bit more english-language to read it as Plugin has Dependencies.